app/src/main/java/com/android/sample/ui/components/TimePicker.kt

```diff
@@ -3,21 +3,27 @@ package com.android.sample.ui.components
 import androidx.compose.runtime.Composable
 import com.google.firebase.Timestamp
 import com.vanpra.composematerialdialogs.MaterialDialog
+import com.vanpra.composematerialdialogs.MaterialDialogState
 import com.vanpra.composematerialdialogs.datetime.time.timepicker
 import com.vanpra.composematerialdialogs.rememberMaterialDialogState
 import java.time.LocalDate
 import java.time.LocalTime
 
 @Composable
-fun MyTimePicker(onTimeSelected: (Timestamp) -> Unit, isOpen: Boolean) {
+fun MyTimePicker(
+    onTimeSelected: (Timestamp) -> Unit,
+    isOpen: Boolean,
+    onCloseRequest: (MaterialDialogState) -> Unit
+) {
   MaterialDialog(
+      onCloseRequest = onCloseRequest,
       dialogState = rememberMaterialDialogState(isOpen),
       buttons = {
         positiveButton(text = "Ok")
         negativeButton(text = "Cancel")
       }) {
         timepicker(
-            initialTime = LocalTime.NOON,
+            initialTime = LocalTime.now().truncatedTo(java.time.temporal.ChronoUnit.MINUTES),
```

Comment:
Very good ! Instead of always starting at noon (LocalTime.NOON), it now starts at the current time